bin/propolis-server/src/main.rs

```diff
@@ -161,6 +213,48 @@ fn build_logger() -> slog::Logger {
     log
 }
 
+fn is_valid_listen_addr_for_dns(listen_addr: IpAddr) -> bool {
+    let IpAddr::V6(addr) = listen_addr else {
+        return false;
+    };
+    addr != Ipv6Addr::LOCALHOST
+}
+
+/// Build metric configuration from the provided registration and listen
+/// addresses.
+///
+/// This will return None if metrics are explicitly disabled.
+fn build_metric_configuration(
+    log: &Logger,
+    metrics_addr: MetricRegistration,
+    listen_addr: IpAddr,
+) -> anyhow::Result<Option<MetricsEndpointConfig>> {
+    let cfg = match metrics_addr {
+        MetricRegistration::Disable => {
+            debug!(
```

Comment:
i feel like i would probably log this at `Info` (at least) --- this seems like the kind of information that i would always want to see when the process starts up at the default log level, and it only gets logged once...

Reply:
Done in 56792ba